docs/src/tutorials/supported_operations.md

```diff
@@ -20,6 +20,16 @@ The specification for the operation files is described in `thin-edge.io` specifi
 
 Supported operations are declared in the cloud specific subdirectory of `/etc/tedge/operations` directory.
 
+#### Supported Operations for Child-Devices
```

Comment:
The placement of this section doesn't look very natural. I'd suggest that we move it further down and place it right before "Working with custom operations"

Reply:
Both were applied (moved child-device section down and left a note at that place).